src/collective/volto/formsupport/restapi/services/submit_form/post.py

```diff
@@ -245,7 +295,7 @@ def filter_parameters(self):
 
     def send_mail(self, msg, encoding):
         host = api.portal.get_tool(name="MailHost")
-        host.send(msg, charset=encoding)
+        host.send(msg, charset=encoding, immediate=True)
```

Comment:
@cekk Why this change was necessary?

Reply:
because without this, i can't catch mail send exceptions like the one from gmail.
I haven't checked it, but it seems that without immediate=True, exceptions are handled by the mailhost and not raised to me (but logged)

Reply:
Okay. But be careful. AFAIK Mailhost is transactional, with your change it is possible that the email will be sent, but for other problems (ConflictError or any other exception) the record will not be saved in the db. This is probably not a problem here. Consider making a comment to explain your choice.

Reply:
ok, comment added